lib/pages/dashboard_page.dart

```diff
@@ -12,9 +12,8 @@ class DashboardPage extends StatefulWidget {
 }
 
 class _DashboardPageState extends State<DashboardPage> {
-  List<String> listItems = ['Items', 'Purchase', 'Sales', 'Report'];
-
   int selectedItem = 0;
+  List<bool> selectedItemList = [true, false, false, false];
```

Comment:
What does this true , false represent?

Reply:
True represents that when we are logged in on the dashboard, so by default, items should be selected. Items is the first menuItem. So it's true.

Reply:
We need to have this implemented it in a slightly better way. If we add more items, this will no longer be readable.